glide-core/redis-rs/redis/src/aio/multiplexed_connection.rs

```diff
@@ -31,6 +31,9 @@ use std::time::Duration;
 #[cfg(feature = "tokio-comp")]
 use tokio_util::codec::Decoder;
 
+// Default connection timeout in seconds
+const DEFAULT_CONNECTION_ATTEMPT_TIMEOUT: Duration = Duration::from_secs(2);
```

Comment:
that's a lot 
since glide and redis-rs are placed together now, can you use the defaults we use for glide? we shouldn't have two defaults for connection timeout

Reply:
can we discuss the default of glide?

Reply:
Changed to 250, will be happy to keep discussing it widely. Checking use cases.